src/libraries/System.Net.Security/src/System/Net/Security/SslStreamPal.Unix.cs

```diff
@@ -3,6 +3,8 @@
 
 using System.Collections.Generic;
 using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
```

Comment:
do we need this? It seems like you only removed ConvertAlpnProtocolListToByteArray bellow, right?

Reply:
Removed